packages/schema/scripts/validate.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
```

Comment:
2024

Reply:
```suggestion
 * Copyright 2024 Google LLC
```
still getting used to it